contracts/staking/contracts/src/fees/MixinExchangeManager.sol

```diff
@@ -45,6 +45,17 @@ contract MixinExchangeManager is
         _;
     }
 
+    /// @dev Adds a new erc20 proxy.
+    /// @param erc20AssetProxy The asset proxy that will transfer erc20 tokens.
+    function addERC20AssetProxy(address erc20AssetProxy)
```

Comment:
Nit - `setERC20AssetProxy` and `emit ERC20AssetProxySet` ~ for consistency w other similar functions in this package.